spec.bs

```diff
@@ -1782,7 +1781,8 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
         1. [=list/For each=] |ig| of |groups|:
           1. [=Batch or fetch trusted bidding signals=] given |trustedBiddingSignalsBatcher|,
-            |ig|, |signalsUrl|, |buyerExperimentGroupId|, |topLevelOrigin|, and |slotSizeQueryParam|.
+            |ig|, |signalsUrl|, |buyerExperimentGroupId|, |topLevelOrigin|, |slotSizeQueryParam|,
+            and |settings|'s [=environment settings object/policy container=].

```

Comment:
It doesn't look like Batch or fetch trusted bidding signals was updated to take a settings object?

Working our way down a couple methods, it looks like "fetch trusted signals" should take one, and set a policy container?  Hrm...  Our V2 dispatcher can actually combined requests across tabs, which doesn't really allow for inheriting a single policy container.  :|

I guess this only covers V1, for now.  Yet more complexity to worry about down the line.

Reply:
yeah this only covers V1 for now. I have no idea with V2 at this moment. We can update this when we spec V2.

Reply:
So, about the rest of my comment - that we aren't passing in the new field to trusted bidding signals requests...?

Reply:
(just replied in the other comment, but copying here) I changed policy container of trusted signals fetches in the begining, but then as you figured that it's ignored when it's cors, so reverted those changes but forgot to revert the call (I can remove this policy container here if we agree on that). I was thinking since it worked the same as not being set (for a CORS request), so may not worth make the change of setting it in the spec.

Reply:
done. Now passing |policyContainer| around and set signal fetch requests' policy container's ip address space this |policyContainer|'s.